internal/providers/terraform/aws/dynamodb_table.go

```diff
@@ -270,10 +264,14 @@ func dataStorageCostComponent(d *schema.ResourceData, u *schema.ResourceData) *s
 
 func continuousBackupCostComponent(d *schema.ResourceData, u *schema.ResourceData) *schema.CostComponent {
 	region := d.Get("region").String()
+	var quantity int64
+	if u != nil && u.Get("monthly_gb_continuous_backup_storage.0.value").Exists() {
+		quantity = u.Get("monthly_gb_continuous_backup_storage.0.value").Int()
+	}
 	return &schema.CostComponent{
 		Name:            "Continuous backup (PITR) storage",
 		Unit:            "GB-months",
-		MonthlyQuantity: decimalPtr(decimal.NewFromInt(u.Get("monthly_gb_continuous_backup_storage.0.value").Int())),
```

Comment:
I think it might be more consistent in infracost if we always put the extra information in brackets last, that might make it simpler for devs to use `jq` or `grep`, so `Continuous backup (PITR) storage` can become `Continuous backup storage (PITR)`. I understand that the AWS cost component name might be `Continuous backup (PITR) storage` but there are other cases where we override the AWS name to make things more understandable/readable.